modules/twinklearv.js

```diff
@@ -35,6 +35,7 @@ Twinkle.arv.callback = function (uid, isIP) {
 	Window.addFooterLink('About SPI', 'WP:SPI');
```

Comment:
I think Wikipedia:Sockpuppet investigations/SPI/Guide to filing cases is probably a more useful link, in line with the others?